### DISCUSSION TITLE
[Bug] Updating scoop fails with error setting certificate file on Windows

### DISCUSSION THREAD

**Discussion by Ziimmie** [ScoopInstaller/Scoop#5668] [Help and Questions]
#### Current Behavior
Whenever I try to update scoop, and most apps with scoop, I get an error saying: `"fatal: unable to access 'https://github.com/ScoopInstaller/Scoop/': error setting certificate file: /usr/ssl/certs/ca-bundle.crt"`. 

#### Expected Behavior
Scoop (and it's apps) should update normally, without any error.

#### Additional context/output
This is odd because I am on Windows, so this path couldn't exist even if I tried to make it. Maybe it's a WSL thing? I also used to have msys2 installed, so maybe that's causing the issue?

### System details

**Windows version:** 11

**OS architecture:** 64bit

**PowerShell version:** Major 5, Minor 1, Build 22621, Revision 1778

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "last_update":  "2023-09-21T16:44:18.2520911-04:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by rashil2000** [comment]
This looks like a problem with your Git installation. Are you able to use other git repositories on your machine normally?

**Comment by danmur** [comment]
This worked for me:

```
git config --global http.sslBackend schannel
```

**Reply by rolambert** [reply]
Thank-you for the workaround. This resolved the issue.

```
~
➜ scoop update
Updating Scoop...
fatal: unable to access 'https://github.com/ScoopInstaller/Scoop.git/': SSL certificate problem: self-signed certificate in certificate chain
Update failed.

~ took 2s
❯ git config --global http.sslBackend schannel

~
➜ scoop update
Updating Scoop...
 * 859d1db51bcc chore(release): Bump to version 0.5.2 (#6080)            6 weeks ago
 * c7ec5c82b23b docs(chglog): Update CHANGELOG for v0.5.2 (#6081)        7 weeks ago
Updating Buckets...
```
